test/Spago/Build/Polyrepo.purs

```diff
@@ -68,97 +71,189 @@ spec = Spec.describe "polyrepo" do
       <> imports
       <> body
 
-    mkMainModuleContent { modName, imports, logStatement, packageName } =
-      mkModuleContent
-        { modName
-        , imports:
-            [ ""
-            , "import Effect (Effect)"
-            , "import Effect.Console (log)"
-            ]
-              <> imports
-        , body:
-            [ ""
-            , "main :: Effect Unit"
-            , "main = do"
-            , "  " <> logStatement
-            ]
-              <>
-                ( packageName # maybe [] \pkgName ->
-                    [ ""
-                    , "packageName :: String"
-                    , "packageName = \"" <> pkgName <> "\""
-                    ]
-                )
-        }
+    mkDependencies :: Array String -> Config.Dependencies
+    mkDependencies = Config.Dependencies <<< Map.fromFoldable <<< map (flip Tuple Nothing <<< mkPackageName)
+
+    -- See `config*` functions for transforms below this binding
+    mkPackageOnlyConfig
+      :: { packageName :: String, srcDependencies :: Array String }
+      -> Array (Tuple String Init.DefaultConfigPackageOptions -> Tuple String Init.DefaultConfigPackageOptions)
+      -> Config.Config
+    mkPackageOnlyConfig initialOptions transforms = do
+      let
+        initConfig :: Tuple String Init.DefaultConfigPackageOptions
+        initConfig = Tuple initialOptions.packageName $
+          { name: mkPackageName initialOptions.packageName
+          , dependencies: initialOptions.srcDependencies
+          , build: Nothing
+          , test: Nothing
+          }
+        finalConfig = foldl (\c f -> f c) initConfig transforms
+      Init.defaultConfig' $ PackageOnly $ snd finalConfig
+
+    configAddSrcStrict :: Tuple String Init.DefaultConfigPackageOptions -> Tuple String Init.DefaultConfigPackageOptions
+    configAddSrcStrict = map \r -> r
+      { build = Just
+          { strict: Just true
+          , censorProjectWarnings: r.build >>= _.censorProjectWarnings
+          , pedanticPackages: r.build >>= _.pedanticPackages
+          }
+      }
```

Comment:
Should these functions go in `Init/Config.purs`?

Reply:
We are using these in tests only so I don't think they should be in the project source tree; at best we might be able to reuse them in other tests outside of the polyrepo stuff, so we can move them to the Test.Prelude or some other common module if we find refactoring opportunities in other tests